src/ui-services/htmlService.js

```diff
@@ -1,28 +1,77 @@
 'use strict';
 
 angular.module("openshiftCommonUI")
-  .factory("HTMLService",
-           function(BREAKPOINTS) {
+  .factory("HTMLService", function(BREAKPOINTS) {
+    var WINDOW_SIZE_XXS = 'XXS';
+    var WINDOW_SIZE_XS = 'xs';
+    var WINDOW_SIZE_SM = 'sm';
+    var WINDOW_SIZE_MD = 'md';
+    var WINDOW_SIZE_LG = 'lg';
+
     return {
+      WINDOW_SIZE_XXS: WINDOW_SIZE_XXS,
+      WINDOW_SIZE_XS: WINDOW_SIZE_XS,
+      WINDOW_SIZE_SM: WINDOW_SIZE_SM,
+      WINDOW_SIZE_MD: WINDOW_SIZE_MD,
+      WINDOW_SIZE_LG: WINDOW_SIZE_LG,
+
       // Ge the breakpoint for the current screen width.
       getBreakpoint: function() {
         if (window.innerWidth < BREAKPOINTS.screenXsMin) {
-          return 'xxs';
+          return WINDOW_SIZE_XXS;
         }
 
         if (window.innerWidth < BREAKPOINTS.screenSmMin) {
-          return 'xs';
+          return WINDOW_SIZE_XS;
         }
 
         if (window.innerWidth < BREAKPOINTS.screenMdMin) {
-          return 'sm';
+          return WINDOW_SIZE_SM;
         }
 
         if (window.innerWidth < BREAKPOINTS.screenLgMin) {
-          return 'md';
+          return WINDOW_SIZE_MD;
+        }
+
+        return WINDOW_SIZE_LG;
+      },
+
+      isWindowSizeOrLess: function(size) {
```

Comment:
size makes me think pixels for some reason, what do you think about something like isWindowBelowBreakpoint and isWindowAboveBreakpoint 

@spadgett @rhamilto opinions?